orchard-cms-video-portal/azuredeploy.json

```diff
@@ -0,0 +1,271 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "appName": {
```

Comment:
If it's end-user facing (i.e. I'll give the url to customers/users) then I agree, if not, generating a name makes the deployment a lot more likely to succeed (and users can always modify the template if they want to change it) - but I'll leave the final call to you...